dtp_apis/create_DTP_API.py

```diff
@@ -245,28 +244,30 @@ def create_action_node(self, task_type, action_node_iri, task_iri, target_as_bui
         if not validators.url(action_node_iri):
             raise Exception("Sorry, the IRI is not a valid URL.")
 
-        payload = json.dumps([
-            {
-                "_classes": [self.DTP_CONFIG.get_ontology_uri('asPerformedAction')],
-                "_domain": self.DTP_CONFIG.get_domain(),
-                "_iri": action_node_iri,
-                "_visibility": 0,
-                self.DTP_CONFIG.get_ontology_uri('hasTaskType'): task_type,
-                self.DTP_CONFIG.get_ontology_uri('processStart'): process_start,
-                self.DTP_CONFIG.get_ontology_uri('processEnd'): process_end,
-                self.DTP_CONFIG.get_ontology_uri('constructionContractor'): contractor,
-                "_outE": [
-                    {
-                        "_label": self.DTP_CONFIG.get_ontology_uri('hasTarget'),
-                        "_targetIRI": target_as_built_iri
-                    },
-                    {
-                        "_label": self.DTP_CONFIG.get_ontology_uri('intentStatusRelation'),
-                        "_targetIRI": task_iri
-                    }
-                ]
-            }
-        ])
+        query_dict = {
+            "_classes": [self.DTP_CONFIG.get_ontology_uri('asPerformedAction')],
+            "_domain": self.DTP_CONFIG.get_domain(),
+            "_iri": action_node_iri,
+            "_visibility": 0,
+            self.DTP_CONFIG.get_ontology_uri('hasTaskType'): task_type,
```

Comment:
shouldn't the hasTaskType be part of the _outE attribute?
I'm not completely sure how exactly you structured everything but it looks like this is still the old version of the hasTaskType which Thomas told us is wrong

Reply:
I see `hasTaskType` in both activity and operation nodes. These types should come from ontology, right? But these types are not added in the `DTP_config` file yet.

Reply:
Fixed in https://github.com/BIM2TWIN-Team/DTP_API/pull/30/commits/f1592d3de790b597f89458e3d5edd58fc2835e22